librz/bin/pdb/pdb_downloader.h

```diff
@@ -49,7 +47,13 @@ void deinit_pdb_downloader(SPDBDownloader *pdb_downloader);
 /**
  * \brief download PDB file
  */
-RZ_API int rz_bin_pdb_download(RzCore *core, PJ *pj, int isradjson, SPDBOptions *options);
+RZ_API int rz_bin_pdb_download(RzBin *bin, PJ *pj, int isradjson, SPDBOptions *options);
+/**
```

Comment:
not sure why these files have the doxygen doc here :( :( we usually track them in the .c files.